test/apicoverage/tools/main.go

```diff
@@ -44,6 +46,23 @@ func getFailedResourceCoverages() *coveragecalculator.CoveragePercentages {
 		ResourceCoverages: percentCoverages}
 }
 
+// Helper function to cleanup any existing Junit XML files.
+// This is done to ensure that we only have one Junit XML file providing the
+// API Coverage summary.
+func cleanupJunitFiles(artifactsDir string) {
+	regexExpr, err := regexp.Compile(`junit_.*\.xml`)
```

Comment:
This should probably be a package variable created with `regexp.MustCompile`

Reply:
Addressed in https://github.com/knative/pkg/pull/719